geomstats/learning/kernel_density_estimation_classifier.py

```diff
@@ -0,0 +1,110 @@
+"""The kernel density estimation classifier on manifolds."""
+
+import math
+
+from sklearn.neighbors import RadiusNeighborsClassifier
+
+import geomstats.backend as gs
+
+
+class KernelDensityEstimationClassifier(RadiusNeighborsClassifier):
+    """Classifier implementing the kernel density estimation on manifolds.
+
```

Comment:
Could you add a description specifying the additional features that our class allow, compared to the `RadiusNeighborsClassifier` scikit-learn class?

`Compared to the RadiusNeighbordsClassifier, this class provide the following additional features:...`

Does it allow new geometries, new kernels? If so, which ones exactly?

Reply:
Also, what is the difference with the class `KernelDensity` from scikit-learn? https://scikit-learn.org/stable/modules/generated/sklearn.neighbors.KernelDensity.html. Maybe add a sentence about this too?

`Compared to the KernelDensity class, ...`

Reply:
Why did you choose to inherit from `RadiusNeighborsClassifier` and not from `KernelDensity`, or from both?